src/components/Dao/DaoPluginsPage.tsx

```diff
@@ -108,24 +106,26 @@ class DaoPluginsPage extends React.Component<IProps, null> {
       <div className={css.wrapper}>
         <BreadcrumbsItem to={`/dao/${daoState.address}/plugins`}>Proposal Plugins</BreadcrumbsItem>
 
-        <Sticky enabled top={50} innerZ={10000}>
-          <h1>Proposal Plugins</h1>
-          { pluginManager ?
-            <TrainingTooltip placement="topLeft" overlay={"A small amount of ETH is necessary to submit a proposal in order to pay gas costs"}>
-              <a className={
-                classNames({
-                  [css.addPluginButton]: true,
-                  [css.disabled]: !pluginManagerActive,
-                })}
-              data-test-id="createProposal"
-              href="#!"
-              onClick={pluginManagerActive ? this.handleNewProposal(pluginManager.id) : null}
-              >
-                Add a Plugin
-              </a>
-            </TrainingTooltip>
-            : ""}
-        </Sticky>
+        <div className={css.paddingTop}>&nbsp;</div>
```

Comment:
does this change related to 3box ?

Reply:
https://github.com/daostack/alchemy/pull/1947#issuecomment-658451101